docs/pages/choose-an-edition/teleport-cloud/architecture.mdx

```diff
@@ -24,15 +24,15 @@ Teleport Enterprise Cloud Proxy does not terminate SSH sessions when using OpenS
 The Cloud Proxy terminates TLS for Application, Database, and Kubernetes sessions.
 
 ## Data retention
-Data retention cannot currently be configured by customers. All Teleport Enterprise Cloud
-customers have audit logs retained in DynamoDB for 1 year, cluster configuration
-retained in DynamoDB indefinitely, and session recordings retained in S3 indefinitely.
-When data retention policies are introduced, customers will be contacted and
-able to specify their preferred data retention schedules.
+All Teleport Enterprise Cloud customers have audit logs retained in S3 for 4 years, cluster
+configuration retained in DynamoDB indefinitely, and session recordings retained in S3 indefinitely.
 
 Customers whose subscriptions lapse will have all session recordings, audit logs, and
 cluster state deleted between 7 and 30 days after the lapse.
 
+Customers can configure "External Audit Storage" to store audit logs and session recordings in their
```

Comment:
I would remove the quotes around External Audit Storage since it's a feature name, not an utterance that we're referring to. I would also add a link to the docs on External Audit Storage so it's clear what we mean.

Reply:
replaced quotes with a link in: e15add6a16